shared/src/ledger/ibc/mod.rs

```diff
@@ -18,6 +18,8 @@ where
     // In ibc-go, u64 like a counter is encoded with big-endian:
     // https://github.com/cosmos/ibc-go/blob/89ffaafb5956a5ea606e1f1bf249c880bea802ed/modules/core/04-channel/keeper/keeper.go#L115
 
+    // TODO: what kinds of write actions are desired for data below?
```

Comment:
These counters don't need to be merklized or diffs

Reply:
Is it ok to encode them with Borsh? It looks like they are encoded a different way when initialized, but when incremented encoded with Borsh

Reply:
oh, yes, it's OK to encode them with Borsh. thanks